CONTRIBUTING.md

```diff
@@ -0,0 +1,96 @@
+# DeepSensor: Contributing Guidelines
+
+🎉🎈🍰 Welcome to DeepSensor repository! 🍰🎈🎉
+
+💫🐥 ☀️We're excited that you're here and want to contribute.☀️ 🐥💫
+
+Thank you for considering a contribution to the DeepSensor project! DeepSensor is a Python package dedicated to modeling environmental data with neural processes. With DeepSensor, we aim to provide a user-friendly, effective, and robust solution for the environmental research community. Our vision is <VISION>. By joining our efforts, you're helping push the boundaries of environmental science and machine learning, making strides towards a clearer understanding of our world.
+
+We want to ensure that every user and contributor feels welcome, included and supported to participate in DeepSensor community. Whether you're a seasoned developer, a machine learning enthusiast, an environmental scientist, or just someone eager to learn and contribute, **you are welcome here**. We value every contribution, be it big or small, and we appreciate the unique perspectives you bring to the project.
+
+We hope that the information provided in this document will make it as easy as possible for you to get involved. If you find that you have questions that are not discussed below, please let us know through one of the many ways to [get in touch](#get-in-touch).
+
+## Inclusivity
+
+We aim to make DeepSensor a collaboratively developed project. We, therefore, require that all our members and their contributions **adhere to our Code of Conduct**. Please familiarize yourself with our Code of Conduct that lists the expected behaviours.
+
+Every contributor is expected to adhere to our Code of Conduct. It outlines our expectations and ensures a safe, respectful environment for everyone.
+
+## Important Resources
+
+If you'd like to find out more about DeepSensor, make sure to check out:
+
+1. **README**: For a project overview, setup, and usage, please refer to our README.
+2. **Project Roadmap**: Familiarize yourself with our direction and goals by checking out the project's roadmap.
+
+## Get in touch
+
+The easiest way to get involved with the active development of DeepSensor is to join our regular community calls. <more info about community calls here>
+
+**Slack Workspace**: Join our DeepSensor Slack channel for discussions, queries, and community interactions.
+
+**Email**: If you prefer a more formal communication method or have specific concerns, please reach us at <ADD EMAIL ADDRESS>.
```

Comment:
I'll add my email address in here, and also change the one in the documentation, for now? Unless you want to change it to your personal email, @tom-andersson ?